pkg/fdbadminclient/mock/admin_client_mock.go

```diff
@@ -595,15 +595,19 @@ func (client *AdminClient) GetExclusions() ([]fdbv1beta2.ProcessAddress, error)
 		return nil, client.mockError
 	}
 
-	pAddrs := make([]fdbv1beta2.ProcessAddress, len(client.ExcludedAddresses))
+	pAddrs := make([]fdbv1beta2.ProcessAddress, 0)
```

Comment:
```suggestion
	pAddrs := make([]fdbv1beta2.ProcessAddress, 0,  len(client.ExcludedAddresses))
```

Any reason to remove the init method with capacity?

Reply:
During debugging I noticed its first parameter is always nil since we are appending new values in the slice.

Reply:
Right, the difference is `make([]fdbv1beta2.ProcessAddress, 0)` vs. (my proposal): `make([]fdbv1beta2.ProcessAddress, 0, len(client.ExcludedAddresses))` the latter creates an empty slice with the capacity of `len(client.ExcludedAddresses)`. Another option would have been to use the index instead of appending.

In general we want to preallocate the capacity if possible.